cloudinit/config/tests/test_users_groups.py

```diff
@@ -46,6 +46,34 @@ def test_handle_users_in_cfg_calls_create_users(self, m_user, m_group):
              mock.call('me2', default=False)])
         m_group.assert_not_called()
 
+    @mock.patch('cloudinit.distros.freebsd.Distro.create_group')
+    @mock.patch('cloudinit.distros.freebsd.Distro.create_user')
+    def test_handle_users_in_cfg_calls_create_users_on_bsd(
+            self,
+            f_user,
+            f_group,
+            l_user,
+            l_group,
+    ):
+        """When users in config, create users with freebsd.create_user."""
+        cfg = {'users': ['default', {'name': 'me2'}]}  # merged cloud-config
+        # System config defines a default user for the distro.
+        sys_cfg = {'default_user': {'name': 'freebsd', 'lock_passwd': True,
+                                    'groups': ['wheel'],
+                                    'shell': '/bin/tcsh'}}
+        metadata = {}
+        cloud = self.tmp_cloud(
+            distro='freebsd', sys_cfg=sys_cfg, metadata=metadata)
+        cc_users_groups.handle('modulename', cfg, cloud, None, None)
+        self.assertItemsEqual(
```

Comment:
No change needed, but I was confused by this method existing, because it was dropped in Python 3's `unittest.TestCase`. We still have access to it, however, because our `CiTestCase` derives from `unittest2.TestCase` which does still retain it.